README.md

```diff
@@ -27,115 +17,94 @@ packages][otel-go], [opentelemetry.io], and the [Go
 quickstart][otel-go-quickstart].
 
 ## An example
-This example pushes metrics to [prometheus](https://prometheus.io/) and tracing to [zipkin](https://zipkin.io/)
 
 ```go
 package main
 
 import (
-	"context"
-	"fmt"
-	"log"
-	"net/http"
-
-	"github.com/bufbuild/connect-go"
-	otelconnect "github.com/bufbuild/connect-opentelemetry-go"
-	// pingv1 is generated from a protobuf schema using protoc-gen-go 
-	pingv1 "github.com/bufbuild/connect-opentelemetry-go/internal/gen/observability/ping/v1"
-	// pingv1connect is generated from a protobuf schema using protoc-gen-connect-go
-	"github.com/bufbuild/connect-opentelemetry-go/internal/gen/observability/ping/v1/pingv1connect"
-	promclient "github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/push"
-	promexport "go.opentelemetry.io/otel/exporters/prometheus"
-	"go.opentelemetry.io/otel/exporters/zipkin"
-	metricsdk "go.opentelemetry.io/otel/sdk/metric"
-	tracesdk "go.opentelemetry.io/otel/sdk/trace"
-)
-
-const (
-	prometheusTarget = "http://localhost:9091"
```

Comment:
https://github.com/bufbuild/connect-opentelemetry-go/pull/29#discussion_r1042668696

> Also, I think it may be more helpful for the example to show a real production setup. Could we configure Prometheus metrics and Jaeger tracing instead?

What ever happened to this idea?

Reply:
To me, there are two reasonable options:

1. We keep the `otelconnect` footprint as small as possible, and explain that the instrumentation uses the OTel globals by default.
2. We configure something real.

I'd prefer (2), but the prometheus and otel APIs involved are so verbose that they make _this_ package seem complex.  Given that, (1) seems better. Using the fussier `WithMeterProvider` and `WithTracerProvider` options to inject a no-op doesn't seem to accomplish either goal - we're guiding users to the fussier entry point to `otelconnect`, but we're not showing them how to configure real data collection.

Let's talk about this in person (shortly).